presto-native-execution/presto_cpp/main/types/tests/RowExpressionTest.cpp

```diff
@@ -278,6 +278,82 @@ TEST_F(RowExpressionTest, varchar3) {
   testConstantExpression(str, "VARCHAR", "\"102\"");
 }
 
+TEST_F(RowExpressionTest, varbinary1) {
+  // The result was generated from
+  // `select to_big_endian_32(1)`.
+  std::string str = R"##(
+        {
+            "@type": "constant",
+            "valueBlock": "DgAAAFZBUklBQkxFX1dJRFRIAQAAAAQAAAAABAAAAAAAAAE=",
+            "type": "varbinary"
+        }
+    )##";
+  auto expected = "\"AAAAAQ==\""; // Base64 value for 1 in big endian.
```

Comment:
Avoid local variable expected and inline the result values in the testConstantExpression line itself for readability. Same comment for the tests below.

Reply:
Updated, thanks!